release-notes/major10/10.6.0/valtimo-backend-libraries.md

```diff
@@ -111,10 +111,26 @@ The following bugs were fixed:
 
   The list columns of Object API objects now displays dates with the configured date format.
 
+* **Objecten API search bugfixes**
+  
+  Several bugs have been fixed for the Objecten API search:
+  - The search now properly searches through nested properties using a different path notation.
+  - It is now possible to search for properties with data type 'boolean'.
+  - It is now possible to search for properties with data type 'date'.
+  - It is now possible to search for properties with data type 'datetime'.
+
 ## Breaking changes
 
 The following breaking changes were introduced:
 
+* **Object search path**
```

Comment:
Sanity check: do we have any other documentation about object searching where we can refer to?

Reply:
No. we're still missing a lot of documentation about object-management

However, this new search path is now properly explained in the tooltips when configuring this